packages/cli/src/net/createFetch.mts

```diff
@@ -64,8 +64,15 @@ function handleFetchError(e: unknown): Promise<Response> {
   }
 
   let message = e.message;
-  if (e.errorName === "Artifacts:ArtifactAlreadyExists") {
-    message = "Version already exists";
+  if (e.errorName === "CannotDeleteDeployedWebsiteVersion") {
+    message = "The site version to delete is live and cannot be deleted";
+    tip = "Run the `site version set` command to change the live site version";
```

Comment:
Probably unset is easier here as it doesn't require a version passed?

Reply:
Yeah we may want to discourage people from using that command as the default though given that it impacts the live website (although we do have a confirmation prompt for the destructive change now)